python/ray/tune/execution/placement_groups.py

```diff
@@ -248,7 +248,9 @@ def __repr__(self) -> str:
         )
 
 
+@PublicAPI(stability="alpha")
```

Comment:
I think this should be DeveloperAPI or private. Actually, I think I prefer private but lets discuss